.history/force-app/main/default/classes/Constants_20210713171109.cls

```diff
@@ -0,0 +1,10 @@
+/**
+ * @description       : class containing global variables
+ * @author            : Mikołaj Lechtański
+ * @last modified on  : 13.07.2021
+**/
+public with sharing class Constants {
```

Comment:
4?

Reply:
dodałem niepotrzebniej jakąś wtyczkę gitową i powrzucało historyczne pliki z folderu ".history". Naprawię to przez .gitignore